### ISSUE TITLE
Sharing of object files between executable builds?

### ISSUE THREAD

**Issue by bos** [haskell/cabal#81]
(Imported from [Trac #89](http://hackage.haskell.org/trac/hackage/ticket/89), reported by bjorn on 2006-07-17)

When building a package which contains multiple executables which share some modules, each module is recompiled for every executable. This can increase the compile time substantially. One problem with compiling each module only once is that different executables can have different compiler options, which can for example affect preprocessing. Maybe modules could be shared if all executables have the same compiler options?

**Comment by jsl** [comment]
Based on the comments, it appears that this issue was fixed in 2010. I propose closing this ticket. Please re-open another issue if this problem still exists. 

/cc @tibbe
